lms/templates/content_item_selection/new_content_item_selection.html.jinja2

```diff
@@ -58,7 +58,10 @@
   "courseId": "{{ course_id }}",
   "formAction": "{{ content_item_return_url }}",
   "formFields": {{ form_fields | tojson }},
+  "googleClientId": "{{ google_client_id }}",
+  "googleDeveloperKey": "{{ google_developer_key }}",
   "lmsName" : "Canvas",
+  "lmsUrl": "{{ lms_url }}",
```

Comment:
This is the URL (at least the origin, I think the path is optional) of the top-level frame in the file picker modal and is needed by the Google Picker for security reasons.